### PR TITLE
docs: show create table

### LINKED ISSUE #324: Update docs for feat: show create table
A document change request is generated from
https://github.com/GreptimeTeam/greptimedb/pull/1336

### PR DESCRIPTION
**Pull request by killme2008**
Adds `SHOW CREATE TABLE` statement

Close #324

### CODE REVIEW

**Review by nicecui — COMMENTED** [review]
~~When will we use this command?~~ 
Got it from https://github.com/GreptimeTeam/greptimedb/issues/535

File: docs/reference/sql/show.md [hunk 1]
```diff
@@ -47,3 +48,36 @@ Show tables by `like` pattern:
 SHOW TABLES like '%prometheus%';
 ```
 
+## SHOW CREATE TABLE
+
+Shows the `CREATE TABLE` statement that creates the named table:
```
[the remark this answers is not in the document]
Comment (killme2008) [reply docs/reference/sql/show.md:53]: 似乎并没有原来这样简洁啊

[the remark this answers is not in the document]
Comment (nicecui) [reply docs/reference/sql/show.md:53]: I was a bit confused about the use cases from the description before, now I understand it from MySql 😂. so the previous sentence is fine. Maybe we need more descriptions about the use case.

**Review by nicecui — APPROVED** [review]

File: docs/reference/sql/show.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (killme2008) [reply docs/reference/sql/show.md:53]: @nicecui  Got it, there is an example below. The user can try to run the sample and get the point.

**Comment by killme2008** [comment]
@xtang PTAL

**Review by xtang — APPROVED** [review]

**Pull request merged**